pyanaconda/ui/tui/spokes/storage.py

```diff
@@ -535,14 +541,13 @@ def indirect(self):
     def refresh(self, args=None):
         NormalTUISpoke.refresh(self, args)
 
-        schemelist = self.partschemes.keys()
-        for i, sch in enumerate(schemelist):
+        scheme_list = self.partschemes.keys()
+        for i, sch in enumerate(scheme_list):
             box = CheckboxWidget(title="%i) %s" %(i + 1, _(sch)), completed=(i == self._selection))
```

Comment:
Could we use the constants from `AUTOPART_CHOICES` to remember the selected scheme instead of the index?